### DISCUSSION TITLE
June 7 outage

### DISCUSSION THREAD

**Discussion by X-Wei** [pockethost/pockethost#215] [Q&A]
Apps cannot log in.

Cannot log in via https://pockethost.io/login

any way to recover the files?

**Comment by Alexk2468** [comment]
Yes same issue for me

**Comment by benallfree** [comment]
Sorry everyone, it’s likely I caused an outage with some housekeeping today. Looking into it now.

**Comment by benallfree** [comment]
Back up 🎸 

Sorry about that everyone, it was completely my fault. I should never touch anything.

@X-Wei @Alexk2468 Please confirm and I'll close this ticket.

**Comment by bloor** [comment]
Ya. Works fine naw.

**Comment by X-Wei** [comment]
is this happening again? All my instances are in idel status, and I cannot go to the admin page.

**Reply by X-Wei** [reply]
> is this happening again? All my instances are in idel status, and I cannot go to the admin page.

now it's working again...
